packages/edit-site/src/components/global-styles/navigation-button.js

```diff
@@ -32,4 +25,32 @@ function NavigationButton( {
 	);
 }
 
-export default NavigationButton;
+function NavigationButton( { path, ...props } ) {
```

Comment:
It's starting to feel like we might want to add a similar component to the UI framework directly.

Reply:
I also felt the same — although we'd need to make sure that the way we define "id"s for each link (and consequently, the selector for restoring focus on them later on) is robust enough for the majority of situations.

Consumers of the component could always decide to go "custom" and create their own component using the `useNavigator` hook, in case the built-in `NavigatorButton` didn't match their needs.

Reply:
I started with creating the `NavigatorLink` and `NavigatorBackLink` components in https://github.com/WordPress/gutenberg/pull/37223/commits/ba08a42afea655a768547cff287c659a602dc363 — I can refine them and add docs/tests if we think the approach is going in the right direction.

Currently `NavigatorLink` renders a `button` by default, and accepts `path` as a prop. Do you think that it would be better to render a `a` tag by default, and use `href` for the prop accepting the path to push?

Reply:
Marking as resolved in favour of other, more recent conversations in this same PR.